modular_skyrat/modules/modular_items/lewd_items/code/lewd_clothing/lewd_shoes.dm

```diff
@@ -1,7 +1,7 @@
 //heels item
 /obj/item/clothing/shoes/latexheels
 	name = "latex heels"
-	desc = "Lace up before use. It's pretty difficult to walk in these."
+	desc = "Lace up before use. It's pretty difficult to walk in these while they are laced."
```

Comment:
already told you on discord but saying here for the record, no readding mechanics to the just mechanic-removed heels